bin/syncProductFlags.ts

```diff
@@ -0,0 +1,43 @@
+import '../src/config';
+import createOrGetConnection from '../src/db';
+
+(async (): Promise<void> => {
+  const createdFromArgument = process.argv[2];
+  const createdToArgument = process.argv[3];
```

Comment:
Idea is to run this script in batches by createdAt to avoid deadlocks on the DB because we will have to update all the rows.

Reply:
I think normally we run it in a stream preferred.
That can handle most things.